frontend/www/js/omegaup/user/profile.ts

```diff
@@ -0,0 +1,170 @@
+import Vue from 'vue';
+import user_Profile from '../components/user/Profile.vue';
+import Highcharts from 'highcharts-vue';
+import { OmegaUp } from '../omegaup';
+import T from '../lang';
+import * as api from '../api';
+import * as ui from '../ui';
+import { types } from '../api_types';
+import { Problem, ContestResult } from '../linkable_resource';
```

Comment:
nit: intentamos que el orden de los imports sea dependencias globales, librerías y componentes, separados por una línea vacía:

```suggestion
import Vue from 'vue';
import Highcharts from 'highcharts-vue';

import { OmegaUp } from '../omegaup';
import T from '../lang';
import * as api from '../api';
import * as ui from '../ui';
import { types } from '../api_types';
import { Problem, ContestResult } from '../linkable_resource';

import user_Profile from '../components/user/Profile.vue';
```